src/Components/Dropdowns/Composed/TypeAheadDropDown.tsx

```diff
@@ -90,19 +96,37 @@ export const TypeAheadDropDown: FC<OwnProps> = ({
   const [shownValues, setShownValues] = useState(items)
```

Comment:
Minor nit, but after this change `shownValues` isn't so semantic anymore and might be confusing. The `shownValues` aren't shown when `largeListValidation` evaluates to true. Maybe we change it to something like `searchResults` or `queryResults` so it's clear that there's a difference between what's being fetched from the list of menu items, versus what's being shown to the user.

Reply:
I think this is a good call

Reply:
Sounds good. I will make this change